subprojects/gradle/build-verdict/src/test/kotlin/com/avito/android/build_verdict/BuildVerdictPluginTest.kt

```diff
@@ -0,0 +1,194 @@
+            errorOutput = "e: /private$temp/app/src/main/kotlin/Uncompiled.kt: (1, 1): Expecting a top level declaration\n" +
+                "e: /private$temp/app/src/main/kotlin/Uncompiled.kt: (1, 11): Expecting a top level declaration"
+        )
+    }
+
+    @Test
+    fun `When kapt fails Then build-verdict contains kapt info`() {
+        generateProject(enableKapt = true) { module ->
+            dir("src/main/kotlin/") {
+                kotlinClass("DaggerComponent", module.packageName) {
+                    """
+                        import dagger.Component
+                        class CoffeeMaker
+                        
+                        @Component
+                        interface DaggerComponent {
+                          fun maker(): CoffeeMaker
+                        }
+                        """.trimIndent()
+                }
+            }
+        }
+
+        val result = gradlew(
+            temp,
+            "assembleDebug",
+            expectFailure = true
+        )
+        result.assertThat()
+            .buildFailed()
+
+        assertBuildVerdict(
+            failedTask = "kaptDebugKotlin",
+            errorOutput = """/private$temp/app/build/tmp/kapt3/stubs/debug/DaggerComponent.java:6: error: [Dagger/MissingBinding] CoffeeMaker cannot be provided without an @Inject constructor or an @Provides-annotated method.
+public abstract interface DaggerComponent {
+                ^
+      CoffeeMaker is requested at
+          DaggerComponent.maker()"""
+        )
+    }
+
+    @Test
+    fun `When unit test fails Then build-verdict contains test info`() {
+        generateProject { module ->
+            dir("src/test/kotlin/") {
+                kotlinClass("AppTest", module.packageName) {
+                    """
+                        import org.junit.Test
+                        import junit.framework.Assert
+                        
+                        class AppTest {
+                        
+                            @Test
+                            fun `test runtime exception`() {
+                                throw RuntimeException()
+                            }
+                            
+                            @Test
+                            fun `test assert true`() {
+                                Assert.assertTrue(false)
+                            }
+                        }
+                                """.trimIndent()
+                }
+            }
+        }
+
+        val result = gradlew(
+            temp,
+            ":app:testDebug",
+            expectFailure = true
+        )
+        result.assertThat()
+            .buildFailed()
+
+        assertBuildVerdict(
+            failedTask = "testDebugUnitTest",
+            errorOutput = """FAILED tests:
+	AppTest.test assert true
+	AppTest.test runtime exception"""
+        )
+    }
+
+    fun `When lint fails Then build-verdict contains lint info`() {
```

Comment:
better mark these stubs with TODO, or even better delete it
i'm still finding your empty stubs(written year ago) in instrumentation plugin :)